### PR TITLE
save image to sdcard

### PR DESCRIPTION
**Pull request by Nghicv**

### CODE REVIEW

File: app/src/main/java/framgia/vn/photo_sketch/bitmaputil/BitmapUtil.java [hunk 1]
```diff
@@ -336,4 +345,25 @@ public static Bitmap flip(Bitmap bitmap, boolean horizontal, boolean vertical) {
...
+            FileOutputStream fos = new FileOutputStream(file, true);
+            BufferedOutputStream bos = new BufferedOutputStream(fos);
+            bitmap.compress(Bitmap.CompressFormat.PNG, 100, fos);
+            bos.flush();
+            bos.close();
+
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+
```
Comment (nvmanh) [inline app/src/main/java/framgia/vn/photo_sketch/bitmaputil/BitmapUtil.java]: blank line

**Comment by cong91** [comment]
OK

File: app/src/main/java/framgia/vn/photo_sketch/bitmaputil/BitmapUtil.java [hunk 2]
```diff
@@ -336,4 +345,25 @@ public static Bitmap flip(Bitmap bitmap, boolean horizontal, boolean vertical) {
...
+        String fileName = FILE_NAME + String.valueOf(System.currentTimeMillis()) + IMAGE_TYPE;
+        File file = new File(dir, fileName);
+        try {
+            FileOutputStream fos = new FileOutputStream(file, true);
+            BufferedOutputStream bos = new BufferedOutputStream(fos);
+            bitmap.compress(Bitmap.CompressFormat.PNG, 100, fos);
+            bos.flush();
+            bos.close();
+
+        } catch (Exception e) {
```
Comment (nvmanh) [inline app/src/main/java/framgia/vn/photo_sketch/bitmaputil/BitmapUtil.java]: exception type

**Comment by phamxuanlu** [comment]
OK

**Comment by nvmanh** [comment]
OK!

**Pull request merged**